docs/features/pixi_global_manifest.md

```diff
@@ -0,0 +1,249 @@
+# Pixi Global Manifest
+
+## Motivation
+
+`pixi global` is currently limited to imperatively managing CLI packages.
+The next iteration of this feature should fulfill the following needs:
+
+- Shareable global environments.
+- Managing complex environments with multiple packages as dependencies
+- Flexible exposure of binaries
+
+## Design Considerations
+
+There are a few things we wanted to keep in mind in the design:
+
+1. **User-friendliness**: Pixi is a user focused tool that goes beyond developers. The feature should have good error reporting and helpful documentation from the start.
+2. **Keep it simple**: The CLI should be all you strictly need to interact with global environments.
+3. **Unsurprising**: Simple commands should behave similar to traditional package managers.
+4. **Human Readable**: Any file created by this feature should be human-readable and modifiable.
+
+## Manifest
+
+The global environments and exposed will be managed by a human-readable manifest.
+This manifest will stick to conventions set by `pixi.toml` where possible.
+Among other things it will be written in the TOML format, be named `pixi-global.toml` and be placed at `~/.pixi/manifests/pixi-global.toml`.
+The motivation for the location is discussed [further below](#multiple-manifests)
+
+```toml title="pixi-global.toml"
+# The name of the environment is `python`
+# It will expose python, python3 and python3.11, but not pip
+[envs.python.dependencies]
+python = "3.11.*"
+pip = "*"
+
+[envs.python.exposed]
+python = "python"
+python3 = "python3"
+"python3.11" = "python3.11"
+
+# The name of the environment is `python_3_10`
+# It will expose python3.10
+[envs.python_3_10.dependencies]
+python = "3.10.*"
+
+[envs.python_3_10.exposed]
+"python3.10" = "python"
+
+```
+
+## CLI
+
+Install one or more packages `PACKAGE` and expose their binaries.
+If `--environment` has been given, all packages will be installed in the same environment.
+If the environment already exists, the command will return with an error.
+`--expose` can be given if `--environment` is given as well or if only a single `PACKAGE` will be installed.
+The syntax for `MAPPING` is `exposed_name=binary_name`, so for example `python3.10=python`.
+
+```
+pixi global install [--expose MAPPING] [--environment ENV] <PACKAGE>...
+```
+
+Remove environments `ENV`.
+```
+pixi global uninstall <ENV>...
+```
+
+Update `PACKAGE` if `--package` is given. If not, all packages in environments `ENV` will be updated.
+If the update leads to binaries being removed, it will offer to remove the mappings.
+If the user declines the update process will stop.
```

Comment:
I think we need to also have some option like --yes, so it is possible to run it in non-interactive mode ( for example in CI ) .

this should be applied also for pixi global remove ( becuase it also have interactivity there )

Reply:
Sounds good, will add `--assume-yes/-y`

Reply:
Ah no, just checked, a simple yes option is not enough here.

I've added a point to open questions

Reply:
Why --yes will not be enough here?

Reply:
Yeah, it might be enough in the end.

I would still prefer if we could think about it more, we don't have to decide on that already.

We could for example add two flags. One for saying yes for deleting mappings, and one for adding.

Reply:
You convinced me, will add the flag